components/org.wso2.carbon.identity.oauth/src/test/java/org/wso2/carbon/identity/openidconnect/DefaultIDTokenBuilderTest.java

```diff
@@ -279,6 +291,13 @@ public void testBuildIDTokenAuthorize() throws Exception {
                 .thenReturn(customClaimsCallbackHandler);
         doNothing().when(customClaimsCallbackHandler).handleCustomClaims(jwtClaimsSet, request);
 
+        mockStatic(AuthorizationGrantCache.class);
+        whenNew(AuthorizationGrantCacheKey.class).withAnyArguments().thenReturn(authorizationGrantCacheKey);
+        when(AuthorizationGrantCache.getInstance()).thenReturn(authorizationGrantCache);
+        when(authorizationGrantCache.getValueFromCacheByToken(any(AuthorizationGrantCacheKey.class))).
+                thenReturn(authorizationGrantCacheEntry);
+        when(authorizationGrantCacheEntry.getEssentialClaims()).thenReturn(null);
```

Comment:
Here we do not return any essential claims, So how can we test auth_time being set as an essential claim?